src/includes/class-alma-wc-settings.php

```diff
@@ -255,11 +255,7 @@ public function get_enabled_plans_definitions() {
 	 * @return string
 	 */
 	public function get_title( $payment_method ) {
-		$title = $this->__get( "title_$payment_method" );
-		if ( ! $title ) {
-			$title = self::default_settings()[ "title_$payment_method" ];
-		}
-		return $title;
+		return Alma_WC_Internationalization::get_field( 'title_' . $payment_method );
```

Comment:
you will pass by `Alma_WC_Internationalization` to check locale and after call `alma_wc_plugin()->settings` for finally access to a local parameter 🤯 

please, import `Alma_WC_Internationalization::get_field` in `Alma_WC_Settings::get_i18n` to avoid this.

Reply:
Done.